minicluster/src/main/java/org/apache/accumulo/cluster/standalone/StandaloneClusterControl.java

```diff
@@ -149,9 +141,8 @@ String getJarFromClass(Class<?> clz) {
 
   @Override
   public void adminStopAll() throws IOException {
-    File confDir = getConfDir();
-    String master = getHosts(new File(confDir, "masters")).get(0);
-    String[] cmd = new String[] {SUDO_CMD, "-u", user, ACCUMULO_CONF_DIR + serverAccumuloConfDir, accumuloPath, Admin.class.getName(), "stopAll"};
+    String master = getHosts(MASTER_HOSTS_FILE).get(0);
+    String[] cmd = new String[] {SUDO_CMD, "-u", user, accumuloPath, Admin.class.getName(), "stopAll"};
```

Comment:
> So you are not OK with the removal of ACCUMULO_CONF_DIR + serverAccumuloConfDir

Oh, no, I'm not because you're still breaking things. I didn't realize you didn't revert this change.

> Why do you need this?

See my previous comment: "StopAll is another utility which requires the correct instance.secret.". It's the same reason I -1'ed the removal of the accumuloServerConf property in the first place..